src/metabase/driver.clj

```diff
@@ -304,7 +304,7 @@
         java.math.BigInteger            :BigIntegerField
         java.sql.Date                   :DateField
         java.sql.Timestamp              :DateTimeField
-        java.util.Date                  :DateField
+        java.util.Date                  :DateTimeField
```

Comment:
because java.util.Date can represent more than just a day, it has the precision to represent down to milliseconds.

this came up specifically in the context of this PR because if you have a mongo value that is something like `"last_login" : ISODate("2014-02-01T10:15:00Z")` it comes back on the driver as java.util.Date and then gets set as `:DateField` which means we are losing precision.